src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,204 @@
+Parallel Integral Curve System
+==============================
+
+Within the VisIt infrastructure is the ability to generate integral curves: streamlines and pathlines. An integral curve is a curve that begins at a seed location X, and is tangent at every point in a vector field. An integral curve is computed by numerical integration of the seed location through the vector field. The image below shows fieldlines through the magnetic field of a core-collapse supernova simulation from the GenASiS code.
+
+IMAGE HERE
+
+The generation of the curves form the basis of VisIt's Parallel Integral Curve System (PICS), made up of the :ref:`Integral Curve operator`, the :ref:`LCS operator`, the :ref:`Limit Cycle operator`, and the :ref:`Poincare operator`. Much of the underlying infrastructure and interface is the same for each operator. The user selects a series of seed locations where curves are generated, which are then visualized and analyzed.
+
+The PICS allows for the computation of Lagrangian Coherent Structures (LCS) using a variety techniques developed by George Haller and his group at ETH Zürich. Haller proposed that the ridges of Finite Time Lyapunov Exponents (FTLE) are indicators of hyperbolic LCS (repelling and attracting). In two dimensional flows, hyperbolic LCS's turn out to be stationary curves of the averaged material shear while elliptic LCS's are stationary curves of the averaged strain. The LCS operator coupled with the Integral Curve (Hyperbolic LCS) and Limit Cycle (Elliptical LCS) operators provide a mechanism to calculate these structures and have been developed in cooperation between the VisIt PICS team and the Haller Group.
+
+INCLUDE IMAGE of elliptical and hyperbolic LCS
+
+For complete details on computing elliptical LCS (closed orbits) and hyperbolic LCS (stretch and shrink lines) see the following paper: `K. Onu, F. Huhn, & G. Haller, LCS Tool: A Computational platform for Lagrangian coherent structures, J. of Computational Science, 7 (2015) 26-36 <http://georgehaller.com/reprints/lcstool.pdf>`_
+
+.. toctree::
+    :maxdepth: 1
+
+    IntegralCurve_operator
+    LCS_operator
+    LimitCycle_operator
+    Poincare_operator
+
+Parameters
+----------
+
+Common to all PICS operators is a four tab GUI: Source, Integration, Appearance, and Advanced (the Poincare operator also has an Analysis tab). These tabs contain many functions that are common across all four operators. The following is a description of those common features. Please note: on a mac system, the contents of the Source and Integration tabs are combined into a single Integration tab.
+
+
+Source
+^^^^^^
+
+The set of points to seed the integral curves in the operator. See specific operator for details.
```

Comment:
You might put the information for all the operators here as a union of all the methods. You can mention that not all the source types apply to all the operators. If the interfaces are not the same for all of them, then perhaps they should be made the same.

Reply:
I took the other approach of putting the intersection of all the methods in this file, and then expanded in the relevant files for each operator. For source type, they all support source types, but the intersection is empty.

Reply:
> If the interfaces are not the same for all of them, then perhaps they should be made the same.

This might be able to be done for things like source type, but it can't be done for all interface objects. I'll make a note of it in the ticket to explore that option.